modules/aleph/includes/aleph.reservation.inc

```diff
@@ -100,7 +85,10 @@ function aleph_reservation_options_submit($account, array $values) {
  */
 function aleph_reservation_default_options($account) {
   // Returns empty array in order to ask every time.
-  return array();
+  return array(
```

Comment:
Would prefer it if the comment first said that we don't have any defaults, and then i could mention that this has the side-effect of always prompting the user for branch/interest period selection.